src/classes/player.js

```diff
@@ -0,0 +1,60 @@
+import {
+  GRAVITY, FRICTION, JUMPHEIGHT, MOVESPEED, CANVAS_HEIGHT,
+} from '../constants';
+
+class Player {
+  constructor() {
+    this.position = {
+      x: 100,
+      y: 100,
+    };
+    this.width = 100;
+    this.height = 100;
+    this.color = 'red';
+    this.velocity = {
+      x: 0,
+      y: 0,
+    };
+    this.isJumping = false;
+    this.isInAir = true;
+    window.addEventListener('keydown', (event) => { this.move(event.key); });
+  }
+
+  step() {
+    if (!(this.position.y + this.height < CANVAS_HEIGHT) && !this.isJumping) {
```

Comment:
Can you add some comments and new lines in this function?
It is kinda hard to read and understand.